release/package.json

```diff
@@ -665,6 +665,25 @@
           "description": "Automatically shows the MSBuild output panel when MSBuild functionality is invoked",
           "type": "boolean"
         },
+        "FSharp.notifications.trace": {
+          "default": false,
+          "description": "Enables more verbose notifications using System.Diagnostics.Activity to view traces from FSharp.Compiler.Service.",
+          "type" : "boolean"
+        },
+        "FSharp.notifications.traceNamespaces": {
+          "default": [
+            "BoundModel.TypeCheck",
+            "BackgroundCompiler."
+          ],
+          "description": "The set of System.Diagnostics.Activity names to watch.",
+          "type" : "array",
+          "required": ["FSharp.notifications.trace"]
+        },
+        "FSharp.openTelemetry.enabled": {
```

Comment:
This setting needs to be true in order for `notifications.trace` to have any effect, right? If so, I'd add that to the description, and then add that that check to the LanguageService CLI arg condition too.

Reply:
No, the notifications uses an `ActivityListener` which is independent of export OTel traces.

Reply:
oh, perfect :+1: